packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerBody.tsx

```diff
@@ -71,10 +72,11 @@ const getStyles = (settings: HighlightedTextSettings) => ({
 
         padding: 0.15rem 0.15rem;
         ${body.small({ fontWeight: 'bold', lineHeight: 'loose' })};
-        font-size: 15px;
-
+        ${until.desktop} {
```

Comment:
Still a little confused, but doesn't the `fontWeight: 'bold'` on [L74](https://github.com/guardian/support-dotcom-components/pull/994/files?diff=split&w=0#diff-a7962cdc2a25505a88c962dbb594c8d4177ad95f2fd44f24a37d0d83987a875bR38) do this?

Reply:
I think font weight : 700 is bold. 800 should make the strong tag extra bold.  Probably best to simplfy and remove, done that now.